docs/loading_data.md

```diff
@@ -0,0 +1,32 @@
+[Back to README](../README.md)
+
+## Loading data
+
+Context and event data are loaded into the MySQL database using a job defined in [dashboard/cron.py](../dashboard/cron.py).
+The job approach leverages the [`django-cron`](https://django-cron.readthedocs.io/en/latest/) library.
+
+### Local development
+
+For local testing, make sure your connection secrets are added (see [Configuration](configuration.md)) and
+your VPN is active. Then, with the application running, run this command in a separate terminal.
+
+```
+docker exec -it student_dashboard /bin/bash -c \
+    "python manage.py migrate django_cron && python manage.py runcrons --force"
+```
+
+After about 30 to 60 seconds, the cron job should have completed and you should have data!
+In the admin interface, there is a table where you can check the status of the cron job runs.
+
+### Cron scheduling for deployment
+
+**Note:** Cron scheduling functionality settings may be removed in the future,
+since this responsibility is often handed off to other infrastructure, like an automation server.
```

Comment:
Optional: I would make this entire line a bold to grab immediate attention

Reply:
I'm gonna leave this as is; I think lots of bold text is hard to read. I can try using quoting to make it stand out a bit more.

Reply:
@pushyamig, what do you think of using quoting to emphasize? For example: https://github.com/ssciolla/my-learning-analytics/blob/issue-661-reorganize-docs/docs/loading_data.md

Reply:
Nice! I just wanted this sentence to grab immediate attention, and whatever you did solved it.